diagnostic_feedback/templates/student/quiz.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n %}
+<div class="diagnostic-feedback diagnostic-feedback-student">
+    {% if user_is_staff %} <a href="#" id="export_data">{% trans "Download Data" %}</a><span id="export_progress"></span> {% endif %}
+    {% if questions %}
+    <div class="msg"><h3></h3></div>
+    <form id="student_view_form">
```

Comment:
`id` attributes are required to be unique within every HTML document, so these hard-coded values will be invalid HTML whenever there is more than one diagnostic xblock on the same page. I think the best fix is just to change these IDs to `class` attributes instead.